internal/cli/events/list_test.go

```diff
@@ -17,7 +17,6 @@
 package events
 
 import (
-	"strings"
```

Comment:
The changes here and in other non streams files were autofixed by the cli lint command... just drive bying them.